workflows/setup-workflows.yml

```diff
@@ -38,6 +38,19 @@ jobs:
         run: |
           rm -rf ckeditor4-workflows-common
 
+      - name: Prepare PR branch
+        if: env.HAS_CHANGES == 1
+        run: |
+          echo "BRANCH_NAME=${{ github.ref }}" >> $GITHUB_ENV
+          if [[ -f "./.github/workflows-config.json" ]]; then
+            pushAsPullRequest=$(jq ".setupWorkflows.pushAsPullRequest" "./.github/workflows-config.json")
+            if [[ "$pushAsPullRequest" == true ]]; then
+              echo "AS_PR=1" >> $GITHUB_ENV
+              echo "BRANCH_NAME=t/setup-workflows-update" >> $GITHUB_ENV
+              git checkout -b t/setup-workflows-update
+            fi
+          fi
```

Comment:
Makes perfect sense. The best approach would be probably extracting config reading logic to separate action (so it can be called with two lines) or adding as external script to be reused.

However, as this is quite simple code I'm for adding it as initial step for now to each workflow which needs it (using copy/paste) and then we can see how problematic it is. I will also add short README section on this.